viz/cmd/uninstall.go

```diff
@@ -86,18 +89,26 @@ func uninstallRunE(ctx context.Context) error {
 	}
 
 	for _, srv := range srvs.Items {
-		if err := deleteResource(srv.TypeMeta, srv.ObjectMeta); err != nil {
+		if err := deleteResource(srv.TypeMeta, srv.ObjectMeta, format); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
-func deleteResource(ty metav1.TypeMeta, meta metav1.ObjectMeta) error {
+func deleteResource(ty metav1.TypeMeta, meta metav1.ObjectMeta, format string) error {
 	r := resource.NewNamespaced(ty.APIVersion, ty.Kind, meta.Name, meta.Namespace)
-	if err := r.RenderResource(os.Stdout); err != nil {
-		return fmt.Errorf("error rendering Kubernetes resource: %w", err)
+	if format == "json" {
+		if err := r.RenderResourceJSON(os.Stdout); err != nil {
+			return fmt.Errorf("error rendering Kubernetes resource: %w", err)
+		}
+		return nil
+	} else if format == "yaml" {
+		if err := r.RenderResourceJSON(os.Stdout); err != nil {
```

Comment:
Should probably call `RenderResource` here, right? Seems like both of them call `RenderResourceJSON`

Reply:
good catch, thanks!